### PR TITLE
feat(world): Implement Logical Height

### PR DESCRIPTION
**Pull request by thelooter**
# Description

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Code follows existing style.
- [x] Unit tests added (if applicable).

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/WorldMock.java [hunk 1]
```diff
@@ -2923,8 +2923,12 @@ public double getHumidity(int x, int y, int z)
 	@Override
 	public int getLogicalHeight()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return switch (environment)
+		{
+			case NETHER, THE_END -> 256;
+            case NORMAL -> 384;
```
Comment (Insprill) [inline src/main/java/be/seeseemelk/mockbukkit/WorldMock.java]: Code style

**Review by Insprill — CHANGES_REQUESTED** [review]

File: src/main/java/be/seeseemelk/mockbukkit/WorldMock.java (hunk 1 above)
Comment (thelooter) [reply src/main/java/be/seeseemelk/mockbukkit/WorldMock.java]: Whats wrong here?

Comment (Insprill) [reply src/main/java/be/seeseemelk/mockbukkit/WorldMock.java]: Spaces not tabs. I was on mobile so it looked like the indentation was wrong, but it should be corrected either way.

**Review by Insprill — APPROVED** [review]
LGTM

**Pull request merged**